<?xml version="1.0" encoding="UTF-8"?>
<!--
/****************************************************************************
**
** This file is part of the Blackfriars Trading Terminal
**
** Copyright (C) 2021 Alexander Mishurov
**
** GNU General Public License Usage
** This file may be used under the terms of the GNU
** General Public License version 3. The licenses are as published by
** the Free Software Foundation and appearing in the file LICENSE.GPL3
** included in the packaging of this file. Please review the following
** information to ensure the GNU General Public License requirements will
** be met: https://www.gnu.org/licenses/gpl-3.0.html.
**
****************************************************************************/
-->
<ui version="4.0">
<class>PrefDialog</class>
<widget class="QDialog" name="PrefDialog">
    <property name="geometry">
        <rect>
            <x>0</x>
            <y>0</y>
            <width>400</width>
            <height>200</height>
        </rect>
    </property>
    <property name="windowTitle">
        <string>Preferences</string>
    </property>
    <layout class="QFormLayout" name="formLayout">

        <!-- Broker -->
        <item row="0" column="0">
            <widget class="QLabel" name="labelBroker">
                <property name="text">
                     <string>Broker</string>
                </property>
            </widget>
        </item>
        <item row="0" column="1">
            <widget class="QComboBox" name="inputBroker"/>
        </item>

        <!-- Credentials -->
        <item row="1" column="0">
            <widget class="QLabel" name="labelCreds">
                <property name="text">
                     <string>Credentials</string>
                </property>
            </widget>
        </item>

        <item row="1" column="1">
            <widget class="QLineEdit" name="inputCreds"/>
        </item>

        <!-- Max Risk -->
        <item row="2" column="0">
            <widget class="QLabel" name="labelDefaultRisk">
                <property name="text">
                     <string>Default Risk</string>
                </property>
            </widget>
        </item>

        <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="inputDefaultRisk">
                <property name="minimum">
                     <number>0</number>
                </property>
                <property name="singleStep">
                    <double>0.25</double>
                </property>
                <property name="maximum">
                     <number>5</number>
                </property>
                <property name="suffix">
                     <string>%</string>
                </property>
            </widget>
        </item>

        <!-- Return / Risk Ratio -->
        <item row="3" column="0">
            <widget class="QLabel" name="labelDefaultTarget">
                <property name="text">
                     <string>Default Target</string>
                </property>
            </widget>
        </item>
        <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="inputDefaultTarget">
                <property name="minimum">
                     <number>0</number>
                </property>
                <property name="singleStep">
                    <double>0.25</double>
                </property>
                <property name="suffix">
                     <string>%</string>
                </property>
            </widget>
        </item>

        <!-- Buttons -->
        <item row="4" column="0" colspan="2">
            <widget class="QDialogButtonBox" name="dialogButtons">
                <property name="standardButtons">
                    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
                </property>
            </widget>
        </item>
    </layout>
</widget>
<resources/>
<connections/>
</ui>
